backend/app/actors/scheduler/plane/PlaneTagParseWorker.scala

```diff
@@ -0,0 +1,192 @@
+            config: ServiceConfiguration,
+            settings: PlaneSettings,
+            projectSettings: PlaneProjectSettings,
+            auth: ServiceAuthentication,
+            projectId: ProjectId): Props =
+    Props(classOf[PlaneTagParseWorker],
+          wsClient,
+          systemServices,
+          config,
+          settings,
+          projectSettings,
+          auth,
+          projectId)
+
+  case object StartParsing
+  case object Parse
+}
+
+class PlaneTagParseWorker(wsClient: WSClient,
+                          systemServices: SystemServices,
+                          config: ServiceConfiguration,
+                          settings: PlaneSettings,
+                          projectSettings: PlaneProjectSettings,
+                          implicit val auth: ServiceAuthentication,
+                          projectId: ProjectId)
+    extends Actor
+    with ActorLogging {
+  import PlaneTagParseWorker._
+
+  var cancellable: Option[Cancellable] = None
+  val apiService      = new PlaneApiServiceImpl(wsClient, config)
+  val defaultParams   = "expand=labels,state,project"
+  val maxResults: Int = projectSettings.maxResults.getOrElse(100)
+
+  val receive: Receive = { case StartParsing =>
+    cancellable = Some(
+      context.system.scheduler.scheduleOnce(0 milliseconds, self, Parse))
+    context.become(parsing)
+  }
+
+  val parsing: Receive = { case Parse =>
+    loadIssues(0, None)
+      .map { result =>
+        // fetched all results, notify
+        if (log.isDebugEnabled) {
+          val keys = result.map(i => s"#${i.id}")
+          log.debug(s"Parsed keys:$keys")
+        }
+        // assemble issue tags
+        val tags = result.map(toPlaneIssueTag)
+        systemServices.tagCache ! TagsUpdated[PlaneIssueTag](
+          projectSettings.planeProjectId,
+          projectId,
+          tags)
+
+      }
+      .andThen { case s =>
+        // restart timer
+        log.debug(s"andThen:restart time $s")
+        cancellable = Some(
+          context.system.scheduler
+            .scheduleOnce(settings.checkFrequency milliseconds, self, Parse))
+      }
+  }
+
+  private def toPlaneIssueTag(issue: PlaneIssue): PlaneIssueTag = {
+
+    val nameTag = projectSettings.tagConfiguration.useTitle match {
+      case false => None
+      case _     => Some(SimpleTag(TagId(issue.name)))
+    }
+
+    val labelTags = projectSettings.tagConfiguration.useLabels match {
+      case false => Seq()
+      case _ =>
+        issue.labels match {
+          case Some(labels) =>
+            labels
+              .filterNot(l =>
+                projectSettings.tagConfiguration.labelFilter.contains(l.name))
+              .map(l => SimpleTag(TagId(l.name)))
+          case None => Seq()
+        }
+    }
+
+    val tags =
+      nameTag.map(t => labelTags :+ t).getOrElse(labelTags)
+
+    val issueLink =
+      s"https://organise.tegonal.com/tegonal-intern/projects/${issue.project.id}/issues/${issue.id}"
```

Comment:
here we need a more dynamic configuration including host and workspace or maybe add somethink like external_base_url

Reply:
Done